File: C/PRAK603-2210817220009-Azka Amalia.c
#include<stdio.h>
int main()
{
    int i, hasil, num, n1, n2;
    scanf("%d %d", &n1,&n2);
    int bil1[n1], bil2[n2];
    if(n1==n2){
        for(i=0;i<n1;i++){scanf("%d", &num);bil1[i]=num;}
        for(i=0;i<n2;i++){scanf("%d", &num);bil2[i]=num;}
        for(i=0;i<n1;i++){hasil=bil1[i]*bil2[i];printf("%d ", hasil);}}
    else{
        printf("Jumlah tidak sama");}
}

File: Python/PRAK603-2210817220009-Azka Amalia.py
n1,n2=map(int,input().split())
if n1==n2:
    nilai1=list(map(int,input().split()))
    nilai2=list(map(int,input().split()))
    mat1=[];mat2=[];num=0
    mat1.append(nilai1[num:num+n1])
    mat2.append(nilai2[num:num+n2])
    for i in range(0,n1):
        print(mat1[0][i]*mat2[0][i],end=' ')
else:
    print("Jumlah tidak sama")